packages/aws-cdk-lib/aws-dynamodb/README.md

```diff
@@ -1,6 +1,704 @@
 # Amazon DynamoDB Construct Library
 
 
+Here is a minimal deployable DynamoDB `GlobalTable` definition:
+
+```ts
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+});
+```
+
+By default, a `GlobalTable` will have one primary replica table in the main deployment region. The properties of the replica table in the main deployment region are configurable via the `GlobalTable` properties. For example, consider the following `GlobalTable` defined in a `Stack` being deployed to `us-west-2`:
+
+```ts
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  contributorInsights: true,
+  tableClass: dynamodb.TableClass.STANDARD_INFREQUENT_ACCESS,
+  pointInTimeRecovery: true,
+});
+```
+
+The above `GlobalTable` definition will result in the provisioning of a single table in `us-west-2` with properties that match what was defined on the `GlobalTable`.
+
+Further reading:
+https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GlobalTables.html
+
+## Keys
+
+When a `GlobalTable` is defined, you must define it's schema using the `partitionKey` (required) and `sortKey` (optional) properties.
+
+```ts
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  sortKey: { name: 'sk', type: dynamodb.AttributeType.NUMBER },
+});
+```
+
+## Contributor Insights
```

Comment:
In terms of paragraph ordering, I think things like billing mode/capacity/gsis/replicas are all much more interesting than contributor insights and deletion protection, so should come first.

Yes, probably these got added in the past by random contributors just looking to satisfy their "must have edited the README" requirement without putting much thought into anything, because in general developers hate writing documentation. They love having it, but *someone else* should be providing them with it! 

Unfortunately, since you're touching this now, that someone else is you today 😅

Reply:
Happy to be that someone else! I've reordered the paragraphs